packages/experimental/ActivityIndicator/src/ActivityIndicator.tsx

```diff
@@ -0,0 +1,95 @@
+  const path = `M${start.x} ${start.y} a${innerRadius} ${innerRadius} 0 1 1 ${innerRadius} ${innerRadius}`;
+
+  return <Path d={path} stroke={color} strokeWidth={width} strokeLinecap="round" />;
+};
+
+export const AnimatedSvg = Animated.createAnimatedComponent(Svg);
+const useStyling = buildUseStyling(stylingSettings);
+export const ActivityIndicator = compose<ActivityIndicatorType>({
+  displayName: activityIndicatorName,
+  ...stylingSettings,
+  slots: {
+    root: AnimatedSvg,
+  },
+  render: (props: ActivityIndicatorProps, useSlots: UseSlots<ActivityIndicatorType>) => {
+    const Slots = useSlots(props);
+    const tokens = useStyling(props).root;
+
+    const memoizedActivityIndicatorData = useMemo(
+      () => ({
+        activityIndicatorColor: props.activityIndicatorColor ? props.activityIndicatorColor : tokens['activityIndicatorColor'],
+        size: props.size ? diameterSizeMap[props.size] : diameterSizeMap[tokens['size']],
+        lineThickness: props.lineThickness
+          ? lineThicknessSizeMap[props.lineThickness]
+          : props.size
+          ? lineThicknessSizeMap[props.size]
+          : lineThicknessSizeMap[tokens['lineThickness']],
+        animating: props.animating != undefined ? props.animating : true,
+        hidesWhenStopped: props.hidesWhenStopped != undefined ? props.hidesWhenStopped : true,
+      }),
+      [],
+    );
+
+    const spinAnimation = useRef(new Animated.Value(0)).current;
+    useEffect(() => {
+      if (memoizedActivityIndicatorData.animating) {
+        Animated.loop(
+          Animated.timing(spinAnimation, {
+            toValue: 359,
+            duration: 750,
+            useNativeDriver: false,
+            easing: Easing.linear,
+          }),
+        ).start();
+      }
+    });
+    const interpolateSpin = spinAnimation.interpolate({
+      inputRange: [0, 359],
+      outputRange: ['0deg', '359deg'],
+    });
+
+    // React Native ActivityIndicator still takes up space when hidden, so to perfectly match would use opacity
+    // hiding opacity makes the screen reader on iOS and Android skip over it
+    const hideOpacity = memoizedActivityIndicatorData.animating == false && memoizedActivityIndicatorData.hidesWhenStopped == true ? 0 : 1;
+
+    const path = getActivityIndicatorPath(
+      memoizedActivityIndicatorData.size,
+      memoizedActivityIndicatorData.lineThickness,
+      memoizedActivityIndicatorData.activityIndicatorColor as string,
+    );
+
+    // props for the AnimatedSvg, which is Slots.root
+    const otherProps = {
+      width: memoizedActivityIndicatorData.size,
+      height: memoizedActivityIndicatorData.size,
+      style: {
+        transform: [{ rotateZ: interpolateSpin }, { perspective: 10 }],
+        opacity: hideOpacity,
+      },
+      accessibilityState: { busy: memoizedActivityIndicatorData.animating },
+    };
+    return (rest: ActivityIndicatorProps) => {
+      const { ...mergedProps } = mergeProps(props, rest);
```

Comment:
I think you can add `otherProps` here to the argument list to just have one set of mergedProps?